compute/test/createComputeHyperdiskFromPool.test.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const {assert} = require('chai');
+const {describe, it} = require('mocha');
+const cp = require('child_process');
+const {DisksClient} = require('@google-cloud/compute').v1;
+const {deleteDisk} = require('./util');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const cwd = path.join(__dirname, '..');
+
+describe('Create compute hyperdisk from pool', async () => {
+  const diskName = 'disk-name';
+  const zone = 'europe-central2-b';
+  const storagePoolName = 'storage-pool-name';
+  const disksClient = new DisksClient();
+  let projectId;
+
+  before(async () => {
+    projectId = await disksClient.getProjectId();
+  });
+
+  after(async () => {
+    await deleteDisk(disksClient, projectId, zone, diskName);
```

Comment:
Hi @gryczj There's no real value in putting the delete logic into a separate module and importing it. I recommend removing `util.js` and just delete the resource here and let it throw; if it throws, the test will fail right here in the `after` hook (and that's a good thing). Thanks!

Reply:
changes applied also to other samples- which were merged yesterday